DataFlowDiagrams/DesigningForSSE.md

```diff
@@ -9,3 +9,57 @@
 ### [Full HTML Report](./DFD-CustomerAccountUpdate.htm)
 
 # Part 2
+## DFD Threat Analysis
+
+The data flow diagrams provide a high-level visualization of how the Elasticsearch software works within a system to move, store, and manipulate data.  We can analyze threats to the system using Microsoft STRIDE threats to identify design-related issues.  The following are the counts of threats mitigated and needs investigation from the automatically identified threats:
+
+## Threats Migitated
+
+|STRIDE|Count|
+|------|-----|
+|Spoofing|2|
+|Tampering|4|
+|Repudiation|10|
+|Information Disclosure|3|
+|Denial of Service|4|
+|Elevation of Privilege|3|
+
+
+## Needs Investigation
+
+|STRIDE|Count|
+|------|-----|
+|Spoofing|3|
+|Tampering|7|
+|Repudiation|1|
+|Information Disclosure|1|
+|Denial of Service|5|
+|Elevation of Privilege|3|
+
+Spoofing could occur in the “Bank Employee” external interactor if a malicious user pretends to be a verified user in order to get sensitive account information.  Tampering with any bits in the system could exist.  For example, if a dataflow contains JSON, JSON processing and hijacking threats may be exploited. Repudiation threats could result from letting everyone write to the logs.  Information disclosure threats appear when looking at who has access to the data in the system and if any of those with access are unauthorized.  Denial of Service threats are related to resource consumption.  Attacks could deadlock the system.  Elevation of Privilege threats could occur if an attacker passes data into the Elasticsearch Replica Node in order to change the flow of program execution.
```

Comment:
I don't think we need to summarize what the STRIDE threats are, I would recommend removing this paragraph unless someone else disagrees.